tools/BUILD

```diff
@@ -100,6 +100,44 @@ py_binary(
     ]
 )
 
+genrule(
+    name = "bazelci_py",
+    srcs = ["@bazelci_py_file//file"],
+    outs = ["bazelci.py"],
+    cmd = "cp $(location @bazelci_py_file//file) $@",
+)
+
+genrule(
+    name = "bcr_presubmit_py",
+    srcs = ["@bcr_presubmit_py_file//file"],
+    outs = ["bcr_presubmit.py"],
+    cmd = "cp $(location @bcr_presubmit_py_file//file) $@",
+)
```

Comment:
What's the reason for these copies? Would it be possible to use `@bazelci_py_file//file` in the `srcs` below?

Reply:
I did try that, the file path in the runfiles will still be the one in the external repo, which doesn't work for proper import with just `import bcr_presubmit`.

Reply:
What if you set `imports = ["."],` for both `py_library` targets?

Reply:
That still doesn't work, I assume `imports = ["."]` resolve to the path of the current package, instead of the source file. For example, what if there are multiple files in srcs from different repo.